src/includes/components/sidebar.php

```diff
@@ -8,12 +8,20 @@
  */
 ?>
 
-<nav class="sidebar open">
+<nav class="sidebar">
```

Comment:
This change on this line of code introduces a regression. The sidebar does not close on click upon page load.